sdks/python/apache_beam/runners/interactive/interactive_runner.py

```diff
@@ -170,8 +170,13 @@ def run_pipeline(self, pipeline, options):
               user_pipeline)):
         streaming_cache_manager = ie.current_env().cache_manager()
         if streaming_cache_manager:
+
+          def exception_handler(e):
+            _LOGGER.error(str(e))
```

Comment:
Same as above. Do we just log and not stop processing?

Reply:
For here, we this is caught at the TestStreamService level and for the same reason, we just log the error so that it's easier for the user to understand. Because it's caught at the service level, it means that we have already stopped processing.